### PR TITLE
Fix inventory edition for decimal

### LINKED ISSUE #4112: (bug) inventory : Editing inventory when interval_de_commande is not an integer
We cannot edit an inventory if the `interval de commande` (`delai de livraison`, `consommation moyenne`) are not an integer

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR is related to #4112 

Close #4112

### CODE REVIEW

**Comment by kwilu** [comment]
Hello @mbayopanda, it seems like you have referenced #4112 in your pull request description, but you have not referenced them in your commit message description(s). When you reference an issue in a commit message, it automatically closes the corresponding issue when the commit is merged.

Please run `git commit --amend` in your command line client to amend your commit message description with `Fixes #4112.`.

An example of a correctly-formatted commit:
```
commit fabd5e450374c8dde65ec35f02140383940fe146
Author: kwilu
Date:   Sat Mar 18 13:42:40 2017 -0700

    pull requests: Check PR commits reference when issue is referenced.

    Fixes #51.
```

Thank you for your contributions to BHIMA!

File: client/src/modules/inventory/list/modals/actions.tmpl.html [hunk 1]
```diff
@@ -156,23 +156,23 @@
...
-        <input class="form-control" type="number" step="1" name="avg_consumption" ng-model="$ctrl.item.avg_consumption">
+        <input class="form-control" type="number" name="avg_consumption" ng-model="$ctrl.item.avg_consumption">
         <span class="input-group-addon" translate>FORM.LABELS.MONTH</span>
       </div>
     </div>
 
     <div class="form-group">
       <label class="control-label" translate>FORM.LABELS.PURCHASE_INTERVAL</label>
       <div class="input-group">
-        <input class="form-control" type="number" step="1" name="purchase_interval" ng-model="$ctrl.item.purchase_interval">
```
Comment (jniles) [inline client/src/modules/inventory/list/modals/actions.tmpl.html:175]: :+1:

**Review by jniles — APPROVED** [review]
LGTM.

**Pull request merged**